source/Octopus.Cli/Commands/Releases/CreateReleaseCommand.cs

```diff
@@ -172,7 +174,11 @@ public async Task Request()
                 release = await Repository.Releases.Create(new ReleaseResource(versionNumber, project.Id, plan.Channel?.Id)
                     {
                         ReleaseNotes = ReleaseNotes,
-                        SelectedPackages = plan.GetSelections()
+                        SelectedPackages = plan.GetSelections(),
+                        VersionControlReferenceResource = new VersionControlReferenceResource
+                        {
+                            GitRef = GitReference
```

Comment:
Can we add a `commandOutputProvider.Information` that prints out the resolved commit hash for VCS releases? just so the user knowns which commit the process came from

Reply:
Good idea:

```
Release created from commit cda2c41ede69222bdf1ffb828a25d02519a0c697 of git reference master.
```